Hunting Queries/SigninLogs/AADSuspectedBruteForce.yaml

```diff
@@ -0,0 +1,38 @@
+id: 056ceb9b-8f07-42b3-853e-ef3779de222e
+name: Suspected Brute force attack Investigation
+description: |
+  'Summarize all the failures and success events for all users in the last 24 hours, 
+  only identify users with more than 100 failures'
+requiredDataConnectors:
+  - connectorId: AzureActiveDirectory
+    dataTypes:
+      - SigninLogs
+  - connectorId: AzureActiveDirectory
+    dataTypes:
+      - AADNonInteractiveUserSignInLogs
+tactics:
+  - CredentialAccess
+relevantTechniques:
+  - T1110
+query: |
+    let authenticationWindow = 24h;
```

Comment:
Remove this, we don't want timeframes embedded in queries when not needed as this breaks the hunting blade UX time slider features.

Reply:
@shainw Please check #3601

Reply:
Removed.